BUILD

```diff
@@ -59,7 +59,7 @@ genrule(
     set -- $(SRCS)
     chmod 755 $$1
     chmod 755 $$2
-    chmod 755 dist/*.objdump
+    chmod 755 -f dist/*.objdump || true
```

Comment:
@snowp changed this internally to something else, we should copy that instead

Reply:
copied